example/simple/feature/a/implementation/src/main/java/com/freeletics/mad/example/feature/a/FeatureADI.kt

```diff
@@ -0,0 +1,14 @@
+package com.freeletics.mad.example.feature.a
+
+import com.freeletics.mad.example.feature.a.nav.FeatureARoute
+import com.freeletics.mad.whetstone.AppScope
+import com.squareup.anvil.annotations.ContributesTo
+import dagger.Module
+
+@Module
+@ContributesTo(FeatureARoute::class)
+object FeatureARetainedModule
+
+@Module
+@ContributesTo(AppScope::class)
+object FeatureASingletonModule
```

Comment:
I'd delete those if they are empty

Reply:
I keep `FeatureARetainedModule` because `FeatureANavigator` scopes to it.

Reply:
Scoping has nothing to do with a module. The scope applies to a component with the same scope and the component is generated in this case